src/api/commands/itemcategory/ItemCategoryUpdateCommand.ts

```diff
@@ -60,29 +58,40 @@ export class ItemCategoryUpdateCommand extends BaseCommand implements RpcCommand
         } as ItemCategoryUpdateRequest);
```

Comment:
const parentItemCategoryId = data.params[3] || 'cat_ROOT';
cat_ROOT is not a categoryId, but a key. This doesn't work.